cmd/argoexec/commands/root.go

```diff
@@ -141,3 +147,12 @@ func checkErr(err error) {
 		panic(err.Error())
 	}
 }
+
+// parseEnvDuration retrieves a duration from the env var specified by `key`. If not set, will return zero duration (0s).
+func parseEnvDuration(key string) (time.Duration, error) {
+	value := os.Getenv(key)
+	if value == "" {
+		return time.Duration(0), nil
+	}
+	return time.ParseDuration(value)
```

Comment:
minor - but you could probably just ignore errors and do `v, _ := time.ParseDuration(os.Getenv(key))`

Reply:
true - the values should be vetted already from the controller :+1: